packages/terra-divider/src/Divider.module.scss

```diff
@@ -1,12 +1,37 @@
 :local {
   .divider {
-    border-bottom-width: 0;
-    border-color: var(--terra-divider-border-color, #dedfe0);
-    border-left-width: 0;
-    border-right-width: 0;
-    border-style: solid;
-    border-top-width: var(--terra-divider-border-top-width, 1px);
-    height: 0;
-    margin: 0;
+    border: 0;
+    height: 1px;
+    line-height: 1em;
+    outline: 0;
+    position: relative;
+    text-align: center;
+
+    &::before {
```

Comment:
I'd like to avoid removing the current variables if possible as that would be non-passive. Instead of setting a background color on the div, I think it would be worthwhile to take a look at https://css-tricks.com/line-on-sides-headers/ and see if we can use the border styles we previously had and apply them to `:before` and `:after` based on the technique in that CSS tricks article.

Reply:
I agree - we should not update the themeing variables - best option is to refactor to with two templates (old hr with no text, plus new div with 3 spans for with text) and use the border trick on new template to satisfy linter rules without any variable changes (and not force exploring a linter override).

Reply:
Updated! [d6ecd19](https://github.com/cerner/terra-core/pull/2296/commits/d6ecd19a5b489194f21b5e4576f27779bf82ca05)

These changes are the amalgamation of @bjankord, @neilpfeiffer, and @JakeLaCombe's suggestions for better accomplishing the intended functionality. Text centering still looks odd, but this seems to at least be going in the right direction.

Reply:
These changes look good to me.